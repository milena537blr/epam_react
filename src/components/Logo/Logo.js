import React, { Component } from 'react';
// import PropTypes from 'prop-types';
import s from "./Logo.module.scss";

class Logo extends Component {
  constructor(props){
    super(props);
  }
  render(){
    return (
      <a className={s.logo} href="#">
        netflixroulette
      </a>
    )
  }
}
Logo.propTypes = {
}

export { Logo };