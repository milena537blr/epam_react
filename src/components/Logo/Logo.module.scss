@import "../../styles/core.scss";

.logo {
  color: $color--red;
  font-weight: bold;
}
