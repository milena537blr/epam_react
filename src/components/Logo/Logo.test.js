import React from "react";
import { mount } from "enzyme";
import { Logo } from "./Logo.js";

test("snapshot test", () => {
  const component = mount(<Logo />);
  expect(component).toMatchSnapshot();
});