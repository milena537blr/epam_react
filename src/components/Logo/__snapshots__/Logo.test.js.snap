// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<Logo>
  <a
    className="logo"
    href="/"
  >
    netflixroulette
  </a>
</Logo>
`;
